easybuild/easyconfigs/j/jq/jq-1.6.eb

```diff
@@ -0,0 +1,30 @@
+name = 'jq'
+version = '1.6'
+
+homepage = 'https://stedolan.github.io/jq/'
+description = """jq is a lightweight and flexible command-line JSON processor. """
+
+toolchain = {'name': 'system', 'version': ''}
+easyblock = 'PackedBinary'
+
+#
+# Example URL: https://github.com/molgenis/ngs-utils/archive/15.12.2.tar.gz
+#
+
+source_urls = [('https://github.com/stedolan/jq/releases/download/jq-%s/' % (version)) ]
+sources = [('jq-%s.tar.gz' % (version))]
+
+checksums = ['5de8c8e29aaa3fb9cc6b47bb27299f271354ebb72514e3accadc7d38b5bbaa72']
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.4'),
+]
```

Comment:
```suggestion
builddependencies = []
```

Reply:
It is a pre-compiled binary: no code will get compiled, so there should not be any builddependencies.